### PR TITLE
Print correct version of bin tools

### PR DESCRIPTION
**Pull request by sudosalim**
Not something important. But if needed, I changed the version to the correct one specified in Cargo.toml. I use these tools for some testing and they print wrong version with --version.

### CODE REVIEW

**Comment by bjorn3** [comment]
https://doc.rust-lang.org/stable/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates:

```rust
let version = env!("CARGO_PKG_VERSION");
```

**Comment by sudosalim** [comment]
> https://doc.rust-lang.org/stable/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates:
> 
> ```rust
> let version = env!("CARGO_PKG_VERSION");
> ```

Thanks, did fix to use it.

File: src/wasm2obj.rs [hunk 1]
```diff
@@ -86,10 +86,11 @@ fn read_wasm_file(path: PathBuf) -> Result<Vec<u8>, io::Error> {
 }
 
 fn main() {
+	let version = env!("CARGO_PKG_VERSION");
```
Comment (bjorn3) [inline src/wasm2obj.rs]: Indentation is off here and in the other files.

Comment (sudosalim) [reply src/wasm2obj.rs]: Sorry, tab messed the indentation. Fixed to use spaces.

**Comment by sunfishcode** [comment]
Looks good to me too!

**Pull request merged**